app.js

```diff
@@ -1,9 +1,38 @@
 const express = require("express");
 const apiRouter = require("./routes/api-router");
-
+const dotenv = require("dotenv");
+const result = dotenv.config();
 const app = express();
+const Twitter = require("twitter");
+
 app.use(express.json());
 
+if (result.error) {
+  console.log(result.error);
+}
+console.log(result.parsed);
+
+const client = new Twitter({
+  consumer_key: process.env.API_KEY,
+  consumer_secret: process.env.API_SECRET_KEY,
+  access_token_key: process.env.ACCESS_TOKEN,
+  access_token_secret: process.env.ACCESS_TOKEN_SECRET
+});
+
+const params = { screen_name: "pillarwallet" };
+
+client.get("statuses/user_timeline", params, function(error, tweets, response) {
+  if (!error) {
+    console.log(tweets);
+  }
+});
```

Comment:
You've executed this in this in the top-level app.js file. Is this not supposed to be in a controller following on from a route?